recipes/SimpleElastix/bld.bat

```diff
@@ -0,0 +1,7 @@
+mkdir build
+cd build
+cmake ../SuperBuild -G "%CMAKE_GENERATOR%"
```

Comment:
You're probably going to want to use nmake otherwise you're going to see build failures for Python 2.7 I think.